web/index.html

```diff
@@ -70,287 +68,350 @@ <h3 class="sub-header">
+                                                        {{ data.index[currentElement][0] }}&nbsp;{{
+                                                        data.index[currentElement][1] }}
+                                                    </div>
+                                                    <div v-else>
+                                                        <table style="font-weight: bold;">
+                                                            <tr v-for="recipe in craftingTreeData">
+                                                                <td class="item"
+                                                                    :class="{'item-used': !saveData.has(recipe.A)}"
+                                                                    @click="navigateTo(recipe.A);"
+                                                                    @mouseover="loadChunk(getChunk(recipe.A))">
+                                                                    {{ data.index[recipe.A][0] }}&nbsp;{{
+                                                                    data.index[recipe.A][1] }}
+                                                                </td>
+                                                                <td>+</td>
+                                                                <td class="item"
+                                                                    :class="{'item-used': !saveData.has(recipe.B)}"
+                                                                    @click="navigateTo(recipe.B);"
+                                                                    @mouseover="loadChunk(getChunk(recipe.B))">
+                                                                    {{ data.index[recipe.B][0] }}&nbsp;{{
+                                                                    data.index[recipe.B][1] }}
+                                                                </td>
+                                                                <td>=</td>
+                                                                <td class="item"
+                                                                    :class="{'item-used': !saveData.has(recipe.C)}"
+                                                                    @click="navigateTo(recipe.C);"
+                                                                    @mouseover="loadChunk(getChunk(recipe.C))">
+                                                                    {{ data.index[recipe.C][0] }}&nbsp;{{
+                                                                    data.index[recipe.C][1] }}
+                                                                </td>
+                                                            </tr>
+                                                        </table>
+                                                        <div v-if="!isMobile && data.costs[currentElement] > 2">
```

Comment:
Added button that appears to let you toggle between full crafting recipes, and crafting recipes based on your imported items. If you have not imported anything, then click on this will also bring you to the import data modal.

Reply:
Make this only appear if save data is loaded, instead of showing import toast

Reply:
@expitau  I honestly disagree with this one, mainly because the Import Data button on it's own in the top corner might be something a lot of people ignore as it could be confused for asking the user to import recipes for our database instead of importing recipes to simplify crafting - by placing a more explicit button exactly where they will be looking when they think "huh I wish it could use my elements so I didn't need to look through this list and figure that out myself" then we actually can make sure that everyone who wants to use it will be able to find it.

If you really hate it, then we can get rid of it, but I think that this is a big enough change that pointing it out to existing users, at least for this next update, and maybe removing it in the future is a better play

Reply:
Nah, this update should be invisible until they import their save data. Even using a bookmarklet can be too technical for some people. We can rename the button to "Import save file" or something to make it more obvious what that does. Feature bloat is annoying for users too, we'll try to stay out of their way as much as possible with this (until I add ads of course hehe)